aws-console-redactor.user.js

```diff
@@ -71,8 +79,14 @@ function replaceByRedactedLink(node) {
  * Copy "data" to clipboard
  */
 function copyToClipboard(data) {
-  navigator.clipboard.writeText(data.trim());
-  alert("Copied to clipboard!");
+  
+  setTimeout(async()=>
```

Comment:
Thanks for this 🙂

How did you come up with this delay?
Can we reduce it to something around 200ms (or maybe use JQuery to wait until the document is in focus)?

Reply:
Actually, after doing the PR I realised that we probably don't need the delay.

What's happening most likely is the `alert()` causes the focus to be lost from document, resulting in the error, while the async operation `writeText()` is completing.